src/api/player/client/teleportToMarker.lua

```diff
@@ -0,0 +1,19 @@
+---teleportToMarker
+---@public

```

Comment:
```suggestion
---@return boolean
---@public
```

Reply:
Oh, exactly it's a good report, i make modif now, thank you :)